wiki/en/en-Contribution.md

```diff
@@ -17,7 +17,7 @@ See the [Jamulus GitHub repo](https://github.com/jamulussoftware/jamulus) for fu
 
 * If you find a mistake, typo or something out of date (in any language) on the website, you can [open an issue on GitHub](https://github.com/jamulussoftware/jamuluswebsite/issues).
 
-* If you think some documentation or information is missing or can be improved, post about that on [the discussion forum](https://github.com/jamulussoftware/jamuluswebsite/discussions) so that it can be discussed first.
```

Comment:
@gilgongo I think this also needs to be fixed in other languages?

Reply:
Unless we can temporarily do it with Google Translate?

ES: el foro de discusión
IT: il forum di discussione
FR: le forum de discussion
DE: das Diskussionsforum

I can put those in on the relevant languages .md on release?

Reply:
German is wrong. Maybe translate the whole sentence?

Reply:
Alternatively just replace the link in `release`, update English in `changes` and accept that the description is slightly incorrect until the next translation.

Reply:
I think the link is still valid. I'll just merge this now.

Reply:
_(the French translation is right)_

Reply:
Spanish is ok too